### ISSUE TITLE
show-on-failure not working correctly

### ISSUE THREAD

**Issue by dakota-maker-by** [toko-bifrost/ms-teams-deploy-card#17]
Hi there, @aldwyn-acn,

When testing the `show-on-failure` configuration that was added, I was still receiving a card notification on a successful run.

My configs are:

```
github-token: ${{ secrets.GITHUB_TOKEN }}
webhook-uri: ${{ secrets.TEAMS_WEBHOOK_URL }}
show-on-start: false
show-on-failure: true
card-layout-exit: complete
```

Any feedback on what I could be doing wrong or if this is a bug would be greatly appreciated

**Comment by aldwyn-acn** [comment]
Hi @dakota-maker-by. Which version did you use? Moreover, may I know where you put the job? If there are any errors on the logs in your Github Actions with this job, it would be helpful. Thanks.

**Comment by dakota-maker-by** [comment]
@aldwyn-acn I am using toko-bifrost/ms-teams-deploy-card@master, so it should pick up the latest version. I put the action right after checkout, and there aren't any errors on the job itself

**Comment by aldwyn-acn** [comment]
Hmm based on the post.ts condition, you have to explicitly set `show-on-exit` to `false`.

**Comment by dakota-maker-by** [comment]
@aldwyn-acn got it. That worked. Thanks so much. I will close out this issue and then possibly add a contrib PR later today if I  can think of an elegant way to make it so just `show-on-failure` will work without the `show-on-exit`

**Comment by aldwyn-acn** [comment]
Hey @dakota-maker-by, apologies for the inconsistency on the docs. I have bumped the fix to 3.1.1. Kindly check.
